src/manager/fvm/mod.rs

```diff
@@ -227,18 +226,14 @@ impl<T: JsonRpcClient + Send + Sync> SubnetManager for LotusSubnetManager<T> {
         subnet: SubnetID,
         gateway_addr: Address,
         from: Address,
-        postbox_msg_cid: Cid,
+        postbox_msg_cid: Vec<u8>,
```

Comment:
Shouldn't we keep it as a `cid` and make it into a `Vec<u8>` for the solidity contract? I think it is confusing to pass here a `Vec<u8>` when it actually is a `Cid` under the hood.

Reply:
In solidity, it's `bytes32` and it's derived from `keccak`. As discussed, we will rename the parameter to `postbox_msg_key` and use `Vec<u8>`.